bin/light-base/src/runtime_service.rs

```diff
@@ -1920,88 +1565,67 @@ impl<TPlat: Platform> Background<TPlat> {
                             }
                         }
                         for to_remove in to_remove {
```

Comment:
can we extract this into a function since it's duplicated here and above?

Reply:
I tried, actually. The code you've highlighted, however, has a bit more logic than the code above, and that extra logic uses variables that are only in this scope. I think that any attempt at de-duplicating this would actually make the code way more complicated.

Reply:
Okay